### PR TITLE
Fix lgtm.com alerts: equality tests

### PR DESCRIPTION
**Pull request by sj**
This PR fixes a number of alerts reported by lgtm.com (@lgtmhq). There is a good number of other alerts well worth looking at here: https://lgtm.com/projects/g/mrdoob/three.js/alerts/

You can set up automated pull request reviews on lgtm.com as well!

### CODE REVIEW

**Comment by mrdoob** [comment]
@lgtmhq is cool! Not sure about your proposed fixes though... I'll have to take a closer look.

File: examples/js/loaders/MMDLoader.js [hunk 1]
```diff
@@ -427,7 +427,7 @@ THREE.MMDLoader.prototype.pourVmdIntoCamera = function ( camera, vmd, name ) {
 
 		var clip = new THREE.AnimationClip( name === undefined ? THREE.Math.generateUUID() : name, -1, tracks );
 
-		if ( clip !== null ) {
```
Comment (mrdoob) [inline examples/js/loaders/MMDLoader.js]: @takahirox can clip be null here?

Comment (takahirox) [inline examples/js/loaders/MMDLoader.js]: Maybe no.... I don't remember why I added this null check logic....
Seems like I should remove.

Comment (takahirox) [inline examples/js/loaders/MMDLoader.js]: I've checked the history.
I used `THREE.AnimationClip.parseAnimation()` which can return `null` before.
Seems like I've forgotten to remove `null` check logic when I replaced it with `new THREE.AnimationClip()`.

Comment (mrdoob) [inline examples/js/loaders/MMDLoader.js]: 👍👍👍

Comment (mrdoob) [inline examples/js/loaders/MMDLoader.js]: @takahirox FYI lgtm.com reports a few more warnings in `MMDLoader`:
https://lgtm.com/projects/g/mrdoob/three.js/snapshot/bf66a7672bac97558f48a8bec4cfc4bcf0382da4/files/examples/js/loaders/MMDLoader.js

**Comment by sjvs** [comment]
@mrdoob @takahirox: thanks for your careful review, let me know if there's anything else I can do to help. 

Would love to hear your feedback on the automated lgtm.com code review for GitHub pull requests. You can enable it here: https://lgtm.com/projects/g/mrdoob/three.js/ci/. That way you get notified of alerts before they get merged into master :)

File: examples/js/loaders/MMDLoader.js (hunk 1 above)
Comment (takahirox) [inline examples/js/loaders/MMDLoader.js]: OK, I'll fix them later!

**Comment by takahirox** [comment]
@sjvs 
Would you please remove clip null check lines from `MMDLoader` in this PR?

**Comment by mrdoob** [comment]
I'll clean the PR myself.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
@sjvs I spent the whole day yesterday fixing 200 alerts in https://lgtm.com/projects/g/mrdoob/three.js/alerts/ The page reported 810 alerts yesterday, today it  reports 937... 🤔

**Comment by sjvs** [comment]
@mrdoob: there was a slight glitch earlier today; things are on their way back to normal. We're in the process of re-analysing the most recent revision of three.js, results should appear soon on https://lgtm.com/projects/g/mrdoob/three.js/alerts/

We've just updated the queries that generate the alerts, which should result in fewer false positives and more true positives. Therefore, the number of alerts may not quite be the same as yesterday's minus 200.

Let me know if you see anything unexpected!

**Comment by mrdoob** [comment]
@sjvs thank you! lovely service by the way!

**Comment by sjvs** [comment]
No problem, @mrdoob - happy to help.

We're very keen to hear feedback from early adopters like yourself, especially regarding the automated reviews of pull requests which we recently introduced. Here's an example of how the guys at NASA's JPL are using it for PRs on one of their open-source projects: https://github.com/Open-MBEE/mdk/pull/105.

The automated review will catch any alerts so you can fix them before merging a PR into master. It'd be great if you could give that a go for Three.js, and let me know your feedback so we can improve :)!

**Comment by mrdoob** [comment]
Yep, just added it. (First time I add anything tot this repo btw 😊)

**Comment by sjvs** [comment]
Great - thanks! Now all we need is a new pull request to see it in action! You're one of the first users, so do let me know if there's anything you'd like to see improved. That's the perk of being an early adopter :wink:

**Comment by mrdoob** [comment]
Will do! 👍

**Pull request merged**